src/Gui/SyntaxHighlighter.cpp

```diff
@@ -33,12 +36,12 @@ class SyntaxHighlighterP
 public:
     SyntaxHighlighterP()
     {
-        cNormalText.setRgb(0, 0, 0); cComment.setRgb(0, 170, 0);
-        cBlockcomment.setRgb(160, 160, 164); cLiteral.setRgb(255, 0, 0);
-        cNumber.setRgb(0, 0, 255); cOperator.setRgb(160, 160, 164);
-        cKeyword.setRgb(0, 0, 255); cClassName.setRgb(255, 170, 0);
-        cDefineName.setRgb(255, 170, 0); cOutput.setRgb(170, 170, 127);
-        cError.setRgb(255, 0, 0);
+        cNormalText = qApp->palette().windowText().color();
+        cComment.setRgb(0, 170, 0); cBlockcomment.setRgb(160, 160, 164);
+        cLiteral.setRgb(255, 0, 0); cNumber.setRgb(0, 0, 255);
+        cOperator.setRgb(160, 160, 164); cKeyword.setRgb(0, 0, 255);
+        cClassName.setRgb(255, 170, 0); cDefineName.setRgb(255, 170, 0);
+        cOutput.setRgb(170, 170, 127); cError.setRgb(255, 0, 0);

```

Comment:
While I appreciate the effort to maintain the original formatting (which normally makes reviews easier and the code more consistent), in this case since you're already touching every line to rearrange for line length, I'd seriously consider reformatting to give each their own line.

Reply:
I hesitated actually. Will push an amended commit on tomorrow.

Reply:
Done